tap_zendesk/streams.py

```diff
@@ -126,7 +132,11 @@ def get_objects(self, start_time):
         '''
         url = self.endpoint.format(self.config['subdomain'])
 
-        for page in http.get_incremental_export(url, self.config['access_token'], start_time):
+        # Set and pass request timeout to config param `request_timeout` value.
+        # If value is 0,"0","" or not passed then it set default to 300 seconds.
+        config_request_timeout = self.config.get('request_timeout')
```

Comment:
@prijendev Same comment here as well

Reply:
In this tap some of the streams using Zenpy sdk. So, to initialize sdk client we have read request_timeout in main() function. While for other streams updated initialization of request_timeout to common parent Stream class to minimize the changes. So, overall we require to read request_timeout 2 times.